docs/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio.md

```diff
@@ -1563,6 +1563,380 @@ properly by the operating system.
 
 1. Pressing the button once should increase the keyboard backlight.
 
+## ECR025.001 Permanent keyboard illumination after cold-boot (Firmware)
+
+**Test description**
+
+This test aims to verify that after enabling permament illumination of keyboard
```

Comment:
Permanent illumination is a constant function, there's no option to turn it on/off.

Reply:
Updated